.github/workflows/patch.yml

```diff
@@ -127,6 +127,8 @@ jobs:
           pyenv global $(pyenv versions | grep '3.10')
           rm -rf ~/frappe-bench/env
           bench -v setup env
+          bench pip install -e ./apps/payments
           bench pip install -e ./apps/erpnext
 
           bench --site test_site migrate
+          bench --site test_site install-app payments
```

Comment:
wondering if we even need this here 😅 

side note: we could once try to move this line to L122 - could probably work & might be more accurate 🤔

Reply:
Nope doesn't work ..the file import payments app is using doesn't exist in v13 (since it had undergone a major refactor in v14)